src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -5,11 +5,19 @@ module sonic-feature{
     namespace "http://github.com/Azure/sonic-feature";
     prefix feature;
 
-    import sonic-types {
-        prefix stypes;
+    description "Feature Table yang Module for SONiC";
+    
+    typedef feature-state {
+        type string {
+            pattern "enabled|disabled|always_enabled|always_disabled";
+        }
     }
 
-    description "Feature Table yang Module for SONiC OS";
+    typedef feature-owner {
+        type string {
+            pattern "kube|local";
```

Comment:
If we could add some description on kube and local, it would be helpful for users.

Reply:
@venkatmahalingam, the description is added `set_owner` leaf where `feature-owner` is used.